lib/fastly/client.rb

```diff
@@ -13,6 +13,7 @@ def initialize(opts)
       @user     = opts.fetch(:user, nil)
       @password = opts.fetch(:password, nil)
       @customer = opts.fetch(:customer, nil)
+      @oldpurge = opts.fetch(:use_old_purge_method, false)  
```

Comment:
IIRC we had some people in networks which didn't allow certain HTTP methods. I could be misremembering.

I agree, I'd like to remove the old method if possible but I think waiting a period of time to deprecate is the nice thing to do.